spec/requests/posts_spec.rb

```diff
@@ -1,17 +1,30 @@
 require 'rails_helper'
+require 'spec_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
+  before :each do
+    @post = Post.new(
+      title: 'Great Efforts yield Great Results',
+      text: 'Its always hard at first put with time you again',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+    @post.save
+
+    describe 'GET /index' do
+      it 'renders the index template' do
+        get '/users/:id/posts/'
+        expect(response.status).to eq(200)
+        expect(response.body).to include('List all users posts')
```

Comment:
- [ ] Kindly make sure to include tests that ensure the correct view template is rendered, as you did in the other action test. e.g `expect(response).to render_template('index')`